### DISCUSSION TITLE
only true programmers will understand this

### DISCUSSION THREAD

**Discussion by IMGROOT2** [For-0/vocabustudy#202] [General]
![image](https://user-images.githubusercontent.com/116324098/198422552-5e4f95cc-6708-436c-aea0-c6de9215c6f9.png)

^ this was in response to the SiddhantBot

**Comment by grimsteel** [comment]
![image](https://user-images.githubusercontent.com/115331356/198427495-80f62052-6df9-49bb-b67f-252bb46bee3a.png)

**Comment by grimsteel** [comment]
![image](https://user-images.githubusercontent.com/115331356/198432996-3772edd9-3db0-477f-a1cb-a3058fea2931.png)
Now we know who the robots are against

**Reply by IMGROOT2** [reply]
Robots: “he’ll be fictional very soon hehehe”

**Reply by IMGROOT2** [reply]
Michael noooooo

**Reply by grimsteel** [reply]
Oh no...

**Comment by PrakmO** [comment]
I understand
